src/docs/product/projects/index.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: Projects
+sidebar_order: 1
+description: "View all the projects you're a member of, then dive into their details quickly."
+---
+A _Project_ connects the error and transaction events in your application to the team members assigned to that project.
+
+You can create a project for a particular language or framework, and typically do so when you first start using Sentry, though you can add projects as you need. For example, you might have separate projects for your API server and frontend client. Setting up multiple projects to reflect your application landscape with finer granularity will go a long way in helping with visibility and triage.
+
+Also, when displaying the **Issue Stream** and **Discover** views, the top-level filter bar considers the projects you are a member of by default. This way, you are looking at information that is immediately relevant to your work.
+
+## Project Homepage
+
+The project homepage provides a high-level overview of all your projects, including:
+
+- All projects of which you're a member
```

Comment:
In Sentry, an account can never be a direct member of a project. Instead, you're member of a team which the project is associated to. Projects can be associated to multiple teams. 

Effectively, the page lists projects by the teams you're a member of. Therefore, a project _can_ appear multiple times on this page.

Reply:
ah, subtle but important. Thanks.